packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -514,15 +530,25 @@ export function mixinNewClient<
 				return {
 					type: "addClient",
 					addedClientId: makeFriendlyClientId(random, clients.length),
+					canBeStashed: options.clientJoinOptions?.stashableClientProbability
+						? random.bool(options.clientJoinOptions.stashableClientProbability)
+						: false,
 				};
 			}
 			return baseOp;
 		};
 	};
 
-	const minimizationTransforms = model.minimizationTransforms as
-		| MinimizationTransform<TOperation | AddClient>[]
-		| undefined;
+	const minimizationTransforms: MinimizationTransform<TOperation | AddClient>[] =
+		(model.minimizationTransforms as
+			| MinimizationTransform<TOperation | AddClient>[]
+			| undefined) ?? [];
+
+	minimizationTransforms.push((op: TOperation | AddClient): void => {
+		if (isClientAddOp(op)) {
+			op.canBeStashed = !op.canBeStashed;
+		}
+	});
```

Comment:
I think you'd want to just set it to be false otherwise it would add `canBeStashed` for repros that don't rely on it, unless that's desired behavior?

```suggestion
	minimizationTransforms.push((op: TOperation | AddClient): void => {
		if (isClientAddOp(op)) {
			op.canBeStashed = false;
		}
	});
```